src/ipc/Port.cc

```diff
@@ -54,6 +55,8 @@ String Ipc::Port::MakeAddr(const char* processLabel, int id)
     assert(id >= 0);
     String addr = channelPathPfx;
     addr.append(service_name.c_str());
+    addr.append('.');
```

Comment:
Please continue to use dashes (`-`) as separators (instead of switching to `.` that has special significance for filenames). Also, if you separate hash from service name (here), then separate it from process label (below) as well.

The same concerns apply to Ipc::Port::CoordinatorAddr() and Ipc::Mem::Segment::GenerateName(), of course.